.github/workflows/_publish.yml_disable

```diff
@@ -0,0 +1,29 @@
+# disabling until we get 2FA working in github
+name: publish
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  tests:
+    uses: ./.github/workflows/tests.yml
+  publish-npm:
+    needs: [tests]
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # Required for OIDC
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v6
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Publish to npm
+        run: | # npm 11.15.1 for OIDC support
+          npm install -g npm@11
```

Comment:
perhaps use a newer node to publish to avoid needing to install an unpinned npm version

Reply:
Yeah, currently still running with mainly 20 for most things and older npm, only recently moved to 11 NPM for OIDC support.